pkg/smb/nodeserver_test.go

```diff
@@ -304,15 +339,16 @@ func TestNodePublishVolume(t *testing.T) {
 }
 
 func TestNodeUnpublishVolume(t *testing.T) {
-	skipIfTestingOnWindows(t)
-	errorTarget := "./error_is_likely_target"
-	targetFile := "./abc.go"
-	targetTest := "./target_test"
+	errorTarget := getTestPath("error_is_likely_target")
+	targetFile := getTestPath("abc.go")
+	targetTest := getTestPath("target_test")
```

Comment:
We already have utility for generating test paths - https://github.com/kubernetes-csi/csi-driver-smb/blob/master/test/utils/testutil/testutil.go#L62

Can we reuse that?

Reply:
In windows the path is supposed to be within the context of the value that was passed as this an argument to `--kubelet-pod-path` of csi-proxy binary The default is `C:\\var\\lib\\kubelet\\`. Since this is a new function, that required within the scope of the file `nodeserver_test.go`, I created a new function.